portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,34 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+/* Creates Slideshow Carousel to go through my projects*/
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+var slideIndex = 1;
+showSlides(slideIndex += n);
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+// Next/previous controls
+function plusSlides(n) {
+  showSlides(slideIndex += n);
 }
+
+// Thumbnail image controls
+function currentSlide(n) {
+  showSlides(slideIndex = n);
+}
+
```

Comment:
Add a doc, describing the function, like what's 'n' suppose to mean? Also change n to a more meaningful name.

Reply:
I added @param for n (which I changed to slideNum) -- is that what you mean by 'add a doc' @hlz03

Reply:
Yes, you can refer https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments
for js style guide. 

In short, the variable name should be meaningful.

Reply:
Done.